src/components/RocketList.js

```diff
@@ -0,0 +1,21 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { fetchRockets } from '../redux/rockets/rocket';
+import Rocket from './Rocket';
+
+const RocketList = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchRockets());
+  }, [dispatch]);
+  const rockets = useSelector((state) => state.rockets);
+  return (
+    <div className="main-container">
+      {rockets.map((rocket) => (
+        <Rocket rocket={rocket} key={rocket.id} />
+      ))}
+    </div>
+  );
+};
+
+export default RocketList;
```

Comment:
Here when this component is rendered, it is making an API request which is not supposed to do, so kindly address the issue so that when a user navigates through the navigation and even if the component is re-rendered the API is not called again.
![image](https://user-images.githubusercontent.com/72081991/208741720-69631fca-271b-4f77-8668-41a1724ca20f.png)
##### kindly, you can use this implementation for reference: [example](https://github.com/yonas44/space-travelers-hub/pull/30/files#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67R14-R16)